src/software/Localization/main_rigLocalizer.cpp

```diff
@@ -386,14 +386,14 @@ int main(int argc, char** argv)
   exporter.initAnimatedCamera("rig");
   exporter.addPoints(localizer->getSfMData().GetLandmarks());
   
-  std::vector<dataio::AlembicExporter> cameraExporters;
+  std::vector<std::unique_ptr<dataio::AlembicExporter> > cameraExporters;
   cameraExporters.reserve(numCameras);
   // this contains the full path and the root name of the file without the extension
   const std::string basename = (bfs::path(exportFile).parent_path() / bfs::path(exportFile).stem()).string();
   for(std::size_t i = 0; i < numCameras; ++i)
   {
-    cameraExporters.emplace_back(basename+".cam"+myToString(i, 2)+".abc");
-    cameraExporters.back().initAnimatedCamera("cam"+myToString(i, 2));
+    cameraExporters.emplace_back( std::unique_ptr<dataio::AlembicExporter>( new dataio::AlembicExporter(basename+".cam"+myToString(i, 2)+".abc")));
```

Comment:
@fabiencastan this is a bit ugly. We could define somewhere `make_unique` if c++14 is not supported. That will look better